uber/models.py

```diff
@@ -744,6 +744,34 @@ def affiliates(self):
                 'total': max(0, amt)
             } for aff, amt in sorted(amounts.items(), key=lambda tup: -tup[1])]
 
+        def insert_test_admin_account(self):
+            """
+            insert a test admin into the database with username "magfest@example.com" password "magfest"
+            this is ONLY allowed if no other admins already exist in the database.
+
```

Comment:
FYI, you've labeled the first param ``session`` even though the parameter name is ``self``.  You generally aren't expected to label the ``self`` parameter in any case, and I suspect this was an artifact of copy/paste that didn't get updated.